src/reducers/reducer.js

```diff
@@ -323,6 +333,16 @@ const initialState = {
     addresses: [],
     error: null,
   },
+  /**
+   * firstAddress {{
+   *   address?: string;
+   *   error?: string;
```

Comment:
```suggestion
   *   address: string | null;
   *   error: string | null;
```

If these are `undefined` (as implied by the `?`) the components will glitch, right?

Reply:
Why not use the `?` notation to indicate a nullable property? We use this notation in other docstrings as well.

When `address` is null and `error` is also null, then the component shows a loading feedback content. It should never glitch.